src/pages/beneficiary/index.tsx

```diff
@@ -101,8 +98,8 @@ const BeneficiaryTable: NextPage<Props> = ({ data }) => {
   );
 };
 
-export async function getStaticProps() {
-  const BASE_URL = config.ITEZ_API_URI;
+export async function getServerSideProps() {
+  const BASE_URL = process.env.ITEZ_API_URI;
```

Comment:
lets change this to use  `SWR` as advised here  and not  serverside props
https://digiandcsh.slack.com/archives/C02JKGD1550/p1637114904084500